app/jbuild

```diff
@@ -3,6 +3,7 @@
 (executable
  ((name biocaml_run_tests)
   (libraries (biocaml_test))
+  (flags (-open Core))
```

Comment:
Shouldn't we add a `:standard` in this line?

Reply:
Yes, I think we should do: `(flags (:standard -short-paths -open Core))`.

Reply:
https://twitter.com/yminsky/status/1011226925069717504